src/serviceDownloadProvider.ts

```diff
@@ -41,20 +41,32 @@ export class ServiceDownloadProvider {
     }
 
     /**
-     * Returns the download url for given platform
+     * Returns the download url for given runtime
      */
-    public getDownloadFileName(platform: Runtime): string {
-        let fileNamesJson = this._config.downloadFileNames;
-        let fileName = fileNamesJson[platform];
-
+    public getDownloadFileName(runtime: Runtime): string {
+        const fileNamesJson = this._config.downloadFileNames;
+        console.log(`Runtimes specified in the configuration file: ${JSON.stringify(fileNamesJson)}`);

```

Comment:
Not sure we should be logging to console, that's just adding a ton of extra bloat we don't really need. Instead I'd suggest adding a new log event emitter that prints out the logs - that way the consumers can pipe it to wherever is needed (and we can include stuff like log level).

Reply:
added the log event as well as a logger and changed all the call to `console.xxx` to use logger. we can add more actual logging if needed in the future.